test/echoErroneousLine.test.cpp

```diff
@@ -0,0 +1,33 @@
+#include "catch.hpp"
+
+#include "utility.hpp"
+
+extern int testNumber;
+
+SCENARIO("echoErroneousLine functions correctly",
+	 "[utility]"){
+  GIVEN("a string with an error"){
+    std::string referenceString1("this is an integer: 123a5");
+    WHEN("there is no newline present"){
+      LOG(INFO) << "Test " << ++testNumber
+                << ":[echoErroneousLine] Errors Expected";
+      auto begin = referenceString1.begin();
+      auto it = std::next( begin, 23 );
+      auto end = referenceString1.end();
+      auto lineNumber = 10l;
+
+      utility::echoErroneousLine( begin, it, end, lineNumber );
```

Comment:
Where are the tests? I was expecting a `REQUIRE` statement or something of its ilk.